src/__tests__/use-telemetry-usage.test.tsx

```diff
@@ -24,7 +24,7 @@ const verifyMetricsAreLoggedOnlyOnce = () => {
 };
 
 const getExpectedMetricName = (lowerCaseComponentName: string) => {
-  return `awsui_${lowerCaseComponentName}_${MetricsTestHelper.formatVersionForMetricName(THEME, PACKAGE_VERSION)}`;
+  return `awsui_${lowerCaseComponentName}_${metricsTestHelper.formatVersionForMetricName(THEME, PACKAGE_VERSION)}`;
```

Comment:
Since it is now a separate module, it can be imported directly, no need to transfer it via test helper.

Metrics test helper need only maintain `resetOneTimeMetricsCache`, because it an internal state. Everything else are pure functions, can be imported/exported directly

Reply:
Makes sense